### DISCUSSION TITLE
How to get RWI associated with a release

### DISCUSSION THREAD

**Discussion by jjcpa** [rfennell/AzurePipelines#1119] [Q&A]
Hi,  for each release there is an associated RWI which is our customized release WI. I checked payload.json and found the only linkage between release and RWI is thru a variable:  
       "RelBuildRWI_ID":{
            "value":"361376",
            "allowOverride":true
         },

Our requirement is to find RWI for the release and generate release note based on the related work on it (circled part)
![Untitled](https://user-images.githubusercontent.com/62727700/139317165-1ce38e32-e024-4677-bc4b-c45bc0565164.png)

Any idea how to achieve it?
Thank you very much
jjcpa

**Comment by rfennell** [comment]
I those WI have been manually associated they will not have been picked up in the past, only ones that were added automatically. 

However, I have just released 3.70.2 which has a new parameter `checkForManuallyLinkedWI`. Does this help?
